Libs/Widgets/ctkTransferFunctionItem.cpp

```diff
@@ -330,3 +332,57 @@ QVariant ctkTransferFunctionItem::itemChange(QGraphicsItem::GraphicsItemChange c
     }
   return res;
 }
+
+//-----------------------------------------------------------------------------
+void ctkTransferFunctionItem::mousePressEvent(QGraphicsSceneMouseEvent* event)
+{
+  QGraphicsObject::mousePressEvent(event);
+
+  ctkTransferFunction* tf = this->transferFunction();
+  if (tf)
+    {
+    ctkTransferFunctionRepresentation* tfRep = tf->representation();
+    Q_ASSERT(tfRep);
+    qreal x = tfRep->mapXFromScene(event->pos().x());
+    qreal y = tfRep->mapYFromScene(event->pos().y());
+    QPointF functionPosition = QPointF(x, y);
+    this->transferFunctionMousePressed(functionPosition, event->button());
+    event->accept();
```

Comment:
Isn't it a problem that now we _always_ consume (accept) the event? (e.g., what if we have multiple transfer functions?)

Maybe instead of consuming the event and emitting a new one, we could install an event filter externally if we need to capture mouse events.

Reply:
mouseMoveEvent is not called for QGraphicsObject unless the mousePressEvent has been accepted.

Reply:
The potential issue that I see is that a ctkTransferFunctionItem would never give a chance to any other graphics items (e.g., other transfer functions or other graphics items) to process mouse move, press, and release events. Have you tried to add two interactive transfer function items to a graphics scene? I suspect that after the proposed change, one would stop working interactively.

Reply:
Removed the signals entirely. Mouse events are now managed from an event filter in Slicer.